src/components/CellViewer.tsx

```diff
@@ -45,21 +44,32 @@ const styles_map_activity_color: StylesMapActivityColorType = {
     [ActivityType.Measurement]: "#ff0000",
 }
 
-const cellFontSize = (cell: VisualArrayCell) =>
-    cell.text.length > 20 ? "14" : cell.text.length > 7 ? "18" : "30"
-
 type CellViewerProps = {
     cell: VisualArrayCell
+    cell_font_size: number
     row_idx: number
     col_idx: number
 }
 
-const CellViewer = ({ cell, row_idx, col_idx }: CellViewerProps) => {
+const CellViewer = ({ cell, cell_font_size, row_idx, col_idx }: CellViewerProps) => {
+    const cellFontSize = (cell: VisualArrayCell) =>
+        cell.text.length > 20
+            ? cell_font_size
+            : cell.text.length > 7
+            ? `${cell_font_size * 1.3}`
+            : `${cell_font_size * 2}`
```

Comment:
Note for a future PR. If you know of a better way to do this go ahead and change it. I know this is how I had it set up